packages/amplify-e2e-tests/src/__tests__/migration/api.connection.migration.test.ts

```diff
@@ -37,14 +37,19 @@ describe('amplify add api', () => {
       amplifyPushUpdate(
         projRoot,
         /Attempting to edit the global secondary index gsi-PostComments on the CommentTable table in the Comment stack.*/,
+        false,
+        false,
+        0,
+        false,
+        true,
```

Comment:
We should make a settings object within amplifyPushUpdate parameters.. I wrote this like this as a quick and dirty solution just to see if this would work.